src/Datadog.Trace/Propagation/MultiplexSpanContextPropagator.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+
+namespace Datadog.Trace.Propagation
+{
+    internal class MultiplexSpanContextPropagator : IPropagator
```

Comment:
We may want to use the name `CompositeTextMapPropagator` for consistency with the OTel .NET SDK.

Reply:
Also in specs https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#composite-propagator

I think we should also start adding doc comments especially to types like this one.